tools/kbcheck/src/data.rs

```diff
@@ -29,13 +29,21 @@ pub fn read_json(path: &Path) -> Result<serde_json::Value, DataError> {
     Ok(serde_json::from_reader(reader)?)
 }
 
+fn is_resolved(entry: &DirEntry) -> bool {
+    entry.path()
+         .to_str()
+         .map(|s| s == "data/resolved")
```

Comment:
You probably want to check if [`s.starts_with("data/resolved")`](https://doc.rust-lang.org/nightly/std/string/struct.String.html#method.starts_with) to handle subdirectories as well.